sdk/storage/storage-blob/src/utils/utils.common.ts

```diff
@@ -66,6 +66,32 @@ export function escapeURLPath(url: string): string {
   return urlParsed.toString();
 }
 
+export function extractPartsWithValidation(connectionString: string): { [key: string]: any } {
```

Comment:
Have you looked into adding a unit test or two for this method?

Reply:
I can add.